### ISSUE TITLE
Bazel Windows upload-artifact fails

### ISSUE THREAD

**Issue by marioecd** [tensorflow/io#945]
I detected this failure during my [PR checks](https://github.com/tensorflow/io/pull/942/checks?check_run_id=651178121). I think it may be unrelated to my changes.

 Run actions/upload-artifact@v10s
##[error]Exit code 1 returned from process: file name 'c:\runners\2.169.1\bin\Runner.PluginHost.exe', arguments 'action "GitHub.Runner.Plugins.Artifact.PublishArtifact, Runner.Plugins"'.
Run actions/upload-artifact@v1
  with:
    name: Windows-bazel-bin
    path: bazel-bin/tensorflow_io
  env:
    BAZEL_VERSION: 3.0.0
    BAZEL_OPTIMIZATION: --copt=-msse4.2 --copt=-mavx --compilation_mode=opt
    pythonLocation: C:\hostedtoolcache\windows\Python\3.7.6\x64
##[error]Path does not exist d:\a\io\io\bazel-bin\tensorflow_io
##[error]Exit code 1 returned from process: file name 'c:\runners\2.169.1\bin\Runner.PluginHost.exe', arguments 'action "GitHub.Runner.Plugins.Artifact.PublishArtifact, Runner.Plugins"'.

**Comment by yongtang** [comment]
Some additional update on why windows build was failing, in case anyone is interested:

In the past we used the default installation of Bazel on GitHub CI and that works fine.

Since Bazel 3.1.0 (or slightly earlier), bazel added a wrapper Bazelisk (or some script) in front of the binary bazel. And the behavior changed. With Bazelist (or front end script), bazel will look for the version string in file `.bazelversion`. Bazel will do exactly the same version matches `.bazelversion`. If there is any mismatch, bazel will through out an error (that was the cause of the windows failure).

The short term fix I added, was to bypass the Bazelisk (or front script). Instead I place the bazel-3.0.0 binary directly to Windows system to get around the issue.

In the future it might makes sense to place a `.bazelversion` for us as well, once we decided to upgrade the bazel on our CI.
